packages/sitecore-jss/src/graphql-request-client.ts

```diff
@@ -46,6 +46,14 @@ export class GraphQLRequestClient {
     query: string | DocumentNode,
     variables?: { [key: string]: unknown }
   ): Promise<T> {
+    const abortController = new AbortController();
```

Comment:
Do we need to create and set abort controller if timeout not specified?

Reply:
What about having a reasonable default timeout (if nothing is specified)? Or just providing by default timeouts set for both endpoints?

Reply:
TBD with @anastasiya29 and the team: seems the current sample application does not support the concept of the default timeout for requests. Adding it will result in to numerous changes in other APIs, so it need to be planned appropriately,